app/api/schema/discount_codes.py

```diff
@@ -166,11 +167,12 @@ def validate_quantity(self, data, original_data):
         DiscountCodeSchemaTicket.quantity_validation_helper(data)
 
         if data.get('tickets_number') and data.get('max_quantity'):
-            if data['tickets_number'] < data['max_quantity']:
-                raise UnprocessableEntityError(
-                    {'pointer': '/data/attributes/tickets-number'},
-                    "tickets-number should be greater than max-quantity",
-                )
+            if data['max_quantity'] >= 0:
+                if data['tickets_number'] < data['max_quantity']:
```

Comment:
and

Reply:
@iamareebjamal I am not sure I follow ? sweat_smile

Reply:
Can be made one statement using and condition

Reply:
ohh sure sure making that change right away!